### PR TITLE
Added --now flag examples in the odo CLI ref.

### PR DESCRIPTION
**Pull request by yhontyk**
- 4.2 and 4.3 branches

Just a small addition to the examples and some fixes (config -> configuration)
@bergerhoffer

### CODE REVIEW

**Comment by boczkowska** [comment]
@bergerhoffer can we put WIP/do not merge for this? The flag should be finished this sprint. I prematurely added the docs. :D

**Comment by bergerhoffer** [comment]
Sure thing @boczkowska! We can add the WIP flag by editing the title of the PR to add "[WIP]" at the front. I'll do that now and will go ahead and peer review, thanks!

File: modules/openshift-developer-cli-basic-odo-cli-commands.adoc [hunk 1]
```diff
@@ -471,6 +477,9 @@ Expose a component to the outside world.
 
   # List the available URLs
   odo url list
+
+  # Create a url in the configuration and apply the changes to the cluster
```
Comment (bergerhoffer) [inline modules/openshift-developer-cli-basic-odo-cli-commands.adoc]: url -> URL

**Review by bergerhoffer — COMMENTED** [review]
Just one minor change, otherwise LGTM!

**Comment by boczkowska** [comment]
@bergerhoffer hi! 
This flag has finally been implemented so we can update the docs now.

File: modules/openshift-developer-cli-basic-odo-cli-commands.adoc [hunk 2]
```diff
@@ -82,9 +85,12 @@ Modify `odo` specific settings within the `config` file.
   odo config set MaxCPU 2
   odo config set CPU 1
   
-  # Set an env variable in the local config
+  # Set an enviroment variable in the local configuration
```
Comment (bergerhoffer) [inline modules/openshift-developer-cli-basic-odo-cli-commands.adoc]: enviroment -> environment

**Comment by bergerhoffer** [comment]
@boczkowska Just one little typo I see. Also, I assume this should also apply for 4.4 now, is that correct?

**Comment by boczkowska** [comment]
@bergerhoffer fixed the typo. Thank you!

Yes, it applies to 4.2, 4.3 and 4.4.

**Comment by bergerhoffer** [comment]
Thanks, looks good now!

**Pull request merged**